src/uct/rocm/copy/rocm_copy_md.c

```diff
@@ -35,12 +35,19 @@ static ucs_config_field_t uct_rocm_copy_md_config_table[] = {
      ucs_offsetof(uct_rocm_copy_md_config_t, rcache),
      UCS_CONFIG_TYPE_TABLE(uct_md_config_rcache_table)},
 
+    {"ENABLE_DMABUF", "n",
+     "Enable using cross-device dmabuf file descriptor",
+     ucs_offsetof(uct_rocm_copy_md_config_t, enable_dmabuf),
+     UCS_CONFIG_TYPE_BOOL},
```

Comment:
maybe better use `UCS_CONFIG_TYPE_TERNARY`, so you will set it to `try` in 1.16 (which means that it will be enabled iff it is supported)

Reply:
we can do that, I would like to have this pr merged in master first in order to bring it over to the 1.15 branch as well with the default being 'no'.. And we can change after that the default for 1.16

Reply:
oh, you mean change it to UCS_CONFIG_TYPE_TERNARY with the default set to 'no' for 1.15, and we change it to 'try' in 1.16?

Reply:
done, thanks!